.mvn/wrapper/maven-wrapper.properties

```diff
@@ -14,6 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-wrapperVersion=3.3.2
+wrapperVersion=3.9.9
```

Comment:
- Where did this come from? I don't see this version at https://github.com/apache/maven-wrapper/releases .